src/main/java/seedu/address/logic/commands/ListRecipeCommand.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+import seedu.address.ui.DisplayedInventoryType;
+
+/**
+ * Lists all recipes in the inventory to the user.
+ */
+public class ListRecipeCommand extends Command {
+
+    public static final String COMMAND_WORD = "listr";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " :lists all recipes";
+
+    public static final String MESSAGE_SUCCESS = "Listed all recipes";
+    public static final String MESSAGE_NO_RECIPES = "You have no recipes in your inventory now, "
+            + "will you like to add some?";
```

Comment:
Might be better to not have "will you like to add some?" if there's no direct button to do so